examples/tutorials/hello-gltf/app.ts

```diff
@@ -69,29 +82,33 @@ export default class AppAnimationLoopTemplate extends AnimationLoopTemplate {
     const far = 2 * this.vantage[0];
     const near = far / 1000;
     const projectionMatrix = new Matrix4().perspective({fovy: Math.PI / 3, aspect, near, far});
+    const cameraTime = this.options['Camera-Animation'] ? time : 0;
```

Comment:
I wonder if something like GLTFAnimator should have a function to generate the matrix? Maybe we set those "options" on that class?

Reply:
I want to have options that are not related to animation too. Also the camera is not part of the gltf animation capabilities, is only for the demo.